src/libraries/System.Security.Cryptography.Pkcs/tests/SignedCms/SignedCmsTests.cs

```diff
@@ -1126,8 +1125,8 @@ public static void EnsureExtraCertsAdded(bool newDocument)
             {
                 cms = new SignedCms();
 
-                // DSA is not supported on mobile Apple platforms, so use ECDsa signed document instead
-                if (PlatformDetection.UsesMobileAppleCrypto)
+                // DSA is not supported on Apple platforms, so use ECDsa signed document instead
+                if (PlatformDetection.UsesAppleCrypto)
```

Comment:
The if isn't really about Apple, it's about "when DSA isn't available".  Rather than chasing down other platforms in the future, let's just use the one DSA lever.

```suggestion
                // When DSA isn't available, use ECDsa instead
                if (!PlatformDetection.IsDSASupported)
```